app/views/support/responsible_bodies/index.html.erb

```diff
@@ -33,7 +33,7 @@
         <td class="govuk-table__cell">
           <%= pluralize(responsible_body.users_who_have_signed_in_at_least_once, "user") %> signed in
         </td>
-        <td class="govuk-table__cell"><%= responsible_body.who_will_order_devices&.humanize %></td>
+        <td class="govuk-table__cell"><%= responsible_body.who_will_order_devices_label&.humanize %></td>
```

Comment:
```suggestion
        <td class="govuk-table__cell"><%= responsible_body.who_will_order_devices_label%></td>
```

`who_will_order_devices_label` already humanizes the string internally, so we don't need to do it again here.

Reply:
👍  thanks